editor/editor/editor.php

```diff
@@ -1119,7 +1122,8 @@ public function getApiActions( $config = [], $context = null ) {
 			'symbolUpdate'               => $pref . Brizy_Admin_Symbols_Api::UPDATE_ACTION,
 			'symbolDelete'               => $pref . Brizy_Admin_Symbols_Api::DELETE_ACTION,
 			'symbolList'                 => $pref . Brizy_Admin_Symbols_Api::LIST_ACTION,
-			'mediaResizeUrl'             => home_url()
+			'mediaResizeUrl'             => home_url(),
```

Comment:
Delete both lines! This is the list of api actions.. not sure why you put this here.